flusher.go

```diff
@@ -44,6 +44,20 @@ func (s *Server) Flush(ctx context.Context) {
 
 	if s.CountUniqueTimeseries {
 		s.Statsd.Count("flush.unique_timeseries_total", s.tallyTimeseries(), []string{fmt.Sprintf("global_veneur:%t", !s.IsLocal())}, 1.0)
+		s.Statsd.Count("flush.unique_timeseries_daily_cumulative", s.tallyTimeseries(), []string{"veneurglobalonly:true"}, 1.0)
```

Comment:
```suggestion
		s.Statsd.Count("flush.unique_timeseries_daily_cumulative", s.cumulativeTimeseries.Estimate(), []string{"veneurglobalonly:true"}, 1.0)
```

I think this is the value you intended to flush?

However, summing counts globally here seems not as good as flushing it as a Set (so that the HLLs can be combined correctly). While the assumptions you reference in the comment below are currently true, they won't be in the future (particularly: host tag on local metrics). I think this is okay to test out for a quick turnaround gut-check, but we probably want to put in the effort to get the more correct data when we merge this.